### PR TITLE
Mark wayland support as experimental

### PR DESCRIPTION
**Pull request by davidedmundson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [![IRC Network](https://img.shields.io/badge/irc-freenode-blue.svg "IRC Freenode")](https://webchat.freenode.net/?channels=sddm)
 
-SDDM is a modern display manager for X11 and Wayland aiming to be fast, simple and beautiful.
+SDDM is a modern display manager for X11 (with experimental Wayland support) aiming to
```
Comment (Vogtinator) [inline README.md]: This sounds like it's also experimental to start Wayland sessions, which is not the case

File: src/common/Configuration.h [hunk 2]
```diff
@@ -78,7 +78,7 @@ namespace SDDM {
         );
 
         Section(Wayland,
-            Entry(CompositorCommand,   QString,     _S("weston --shell=fullscreen-shell.so"),   _S("Path of the Wayland compositor to execute when starting the greeter"));
+            Entry(CompositorCommand,   QString,     _S("weston --shell=fullscreen-shell.so"),   _S("Path of the Wayland compositor to execute when starting the greeter (Experimental)"));
```
Comment (Vogtinator) [inline src/common/Configuration.h]: This option isn't really experimental, only `DisplayServer=wayland` is

**Review by Vogtinator — DISMISSED** [review]

**Review by hsitter — APPROVED** [review]
lgtm

**Review by Vogtinator — COMMENTED** [review]
Retroactive +1

**Comment by Vogtinator** [comment]
(Interesting how GitHub still shows that a review is pending but I can't approve it - only comment)

**Pull request merged**